libs/api/domains/health-directorate/src/lib/health-directorate.resolver.ts

```diff
@@ -0,0 +1,60 @@
+import { Args, Mutation, Query, Resolver } from '@nestjs/graphql'
+
+import { UseGuards } from '@nestjs/common'
+
+import {
+  CurrentUser,
+  IdsUserGuard,
+  Scopes,
+  ScopesGuard,
+} from '@island.is/auth-nest-tools'
+import type { User } from '@island.is/auth-nest-tools'
+import { ApiScope } from '@island.is/auth/scopes'
+import {
+  DonationException,
+  DonorStatus,
+  DonorStatusInput,
+} from './models/organ-donation.model'
+import type { Locale } from '@island.is/shared/types'
+import { Vaccinations } from './models/vaccinations.model'
+import { HealthDirectorateService } from './health-directorate.service'
+
+@UseGuards(IdsUserGuard, ScopesGuard)
+@Scopes(ApiScope.internal)
+@Resolver()
```

Comment:
Maybe we should add audit here?

```
@Audit({ namespace: '@island.is/api/health-directorate' })

...


  @Audit()
```

Reply:
Yebb! 💯 I just forgot about it existing 🙈